pkg/clients/aws.go

```diff
@@ -382,6 +383,15 @@ func UsePodServiceAccountAssumeRoleWithWebIdentity(ctx context.Context, _ []byte
 	return &cnf, err
 }
 
+const webIdentityTokenFileDefaultPath = "/var/run/secrets/eks.amazonaws.com/serviceaccount/token"
```

Comment:
I think we should move this up in this file with other consts.

Reply:
I would prefer to keep it together with code block where it is used for better readability since the definition order does not matter in Go.